controllers/protectedController.js

```diff
@@ -0,0 +1,19 @@
+import { ProtectedRepository } from "../repositories/protectedRepository.js";
+
+export class ProtectedController {
+  constructor() {
+    this.protectedRepository = new ProtectedRepository();
+  }
+
+  getProtected = async (req, res, next) => {
+    console.log("getProtected")
+
+    try {
+      const response = await this.protectedRepository.getProtected();
+      res.status(200).json(response);
+    } catch (error) {
+      res.status(401).json({ message: "UnathorizedOrAuthHeadersMissing" });
```

Comment:
Well, it looks to me as 2 separate errors that must be handled separately. So either use 1 generic response or handle them separately